warehouse/models/mart/gtfs_quality/fct_daily_rt_feed_guideline_checks.md

```diff
@@ -0,0 +1,14 @@
+{% docs fct_daily_rt_feed_guideline_checks %}
+
+Each row represents a date/guideline check/feed combination, with pass/fail
+information indicating whether that feed complied with that check on that date.
+A row will exist for every check, for every row from the index which is driven
+by fct_daily_rt_feeds. Only contains checks that are performed at the feed
+level.
+
+Here is a list of currently-implemented checks:
+
+| Check | Feature | Description |
+| ------------------------------------ |---------|------------ |
+|No critical errors in the MobilityData GTFS Realtime Validator | Compliance | The feed has at least one GTFS-RT file present on the given day, and GTFS Realtime Validator produced no critical errors for any RT feed on that day.|
```

Comment:
`base64_url` differs by feed type, so this is not aggregating across feed types (I think "any RT feed" is wrong here).

Reply:
Ah yeah let me clarify since this isn't rolled up to the quartet yet.